guides/common/modules/proc_using-red-hat-insights.adoc

```diff
@@ -57,6 +57,21 @@ You can manually synchronize the recommendations using the following procedure:
 
 If you have not set up the API token, you are prompted to create one before using this page.
 
+.Removing hosts from Inventory of Red{nbsp}Hat Insights when they are removed from {Project}
+When hosts are removed from {Project}, they can also be removed from the inventory of Red{nbsp}Hat Insights, either automatically or manually.
+To configure it automatically, perform the following steps:
+
+. In the {ProjectWebUI}, navigate to *Configure* > *Inventory Upload*.
+. Enable the *Automatic Mismatch Deletion* setting.
+
+[NOTE]
+====
+You must have administrator privilege for the organization to enable this setting.
```

Comment:
We should probably list the required permissions in particular here.

Reply:
Agree and Addressed :+1:

Reply:
Adding prerequisite and procedure will mix with the heading. It is creating confusion.

Reply:
And use the following text in the Note:
`Your user account has a role assigned that grants the `view_settings` and `edit_settings` permissions.`

Reply:
Added :+1: